pkg/export/otel/traces.go

```diff
@@ -74,9 +74,11 @@ func newTracesReporter(svcName, endpoint string) (*TracesReporter, error) {
 		return nil, fmt.Errorf("creating trace exporter: %w", err)
 	}
 
+	bsp := trace.NewBatchSpanProcessor(r.traceExporter, trace.WithMaxExportBatchSize(4096), trace.WithMaxQueueSize(4096))
```

Comment:
I'll merge this now to have a checkpoint to start working today, and I will make these values configurable by the user.